pkg/healthcheck/healthcheck.go

```diff
@@ -0,0 +1,25 @@
+package healthcheck
```

Comment:
Please add a one-line description of this package.

Reply:
fixed